docs/fundamentals/runcard.rst

```diff
@@ -1,2 +1,387 @@
+                    offset_i: 0. # -0.012
+                    offset_q: 0.
+                    num_bins: 1
+                    intermediate_frequency: 10.e+06
+                    gain_imbalance: 0.940
+                    phase_imbalance: 14.482
+                    hardware_modulation: true
+                -   identifier: 1
+                    chip_port_id: drive_line_q1
+                    output_i: 2
+                    output_q: 3
+                    gain_i: 1
+                    gain_q: 1
+                    offset_i: 0
+                    offset_q: 0
+                    num_bins: 1
+                    intermediate_frequency: 20.e+06
+                    gain_imbalance: 0.5
+                    phase_imbalance: 0
+                    hardware_modulation: true
+        -   name: QCM
+            alias: QCM1
+            firmware: 0.7.0
+            num_sequencers: 4
+            out_offsets: [0.0, 0.0, 0.0, 0.0]
+            awg_sequencers:
+                -   identifier: 0
+                    chip_port_id: flux_line_q0
+                    output_i: 0
+                    output_q: 1
+                    gain_i: 0.1
+                    gain_q: 0.1
+                    offset_i: 0.
+                    offset_q: 0.
+                    num_bins: 1
+                    intermediate_frequency: 10.e+06
+                    gain_imbalance: .5
+                    phase_imbalance: 0.
+                    hardware_modulation: true
+                -   identifier: 1
+                    chip_port_id: flux_line_q1
+                    output_i: 1
+                    output_q: 0
+                    gain_i: 1
+                    gain_q: 1
+                    offset_i: 0
+                    offset_q: 0
+                    num_bins: 1
+                    intermediate_frequency: 0.
+                    gain_imbalance: 0.5
+                    phase_imbalance: 0
+                    hardware_modulation: true
+        -   name: rohde_schwarz
+            alias: rs_1
+            firmware: 4.2.76.0-4.30.046.295
+            power: 16
+            frequency: 8.0726e+09
+            rf_on: true
+        -   name: mini_circuits
+            alias: attenuator
+            firmware: None
+            attenuation: 32
+
+    instrument_controllers:
+        -   name: qblox_cluster
+            alias: cluster_controller_0
+            reference_clock: internal
+            connection:
+            name: tcp_ip
+            address: 192.168.1.20
+            modules:
+                -   alias: QRM1
+                    slot_id: 12
+                -   alias: QCM-RF1
+                    slot_id: 6
+                -   alias: QCM1
+                    slot_id: 14
+                -   alias: QCM2
+                    slot_id: 14
+        -   name: rohde_schwarz
+            alias: rohde_schwarz_controller_0
+            reference_clock: internal
+            connection:
+            name: tcp_ip
+            address: 192.168.1.11
```

Comment:
make sure the ips are not the same as the actual ones, for security reasons?

Reply:
we could also change the names of the instruments to something more general (i.e. awg_cluster) instead of saying the name of the vendors

Reply:
Nice thinking, I'll change the IPs and the vendors, that I see easy to do so!